manifest/olm/crd/sparkclusteroperator.0.3.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sparkoperator.v0.3.3
+  namespace: placeholder
+  annotations:
+    categories: apache spark, apache, spark
+    certified: false
```

Comment:
Should be the string `"false"`

Reply:
ty :+1:

Reply:
The change is needed to work with Operator Hub

Reply:
The change is needed to work with Operator Hub